src/containers/MyNdla/Folders/FolderSharedPage.tsx

```diff
@@ -0,0 +1,222 @@
+      ),
+    };
+
+    const actions = [copyFolder];
+
+    if (showSaveLinkButton) {
+      actions.push(saveLink);
+    }
+
+    return <SettingsMenu menuItems={actions} />;
+  }, [folder, showSaveLinkButton]);
+
+  useEffect(() => {
+    if (!sharedFolders) return;
+    setShowSaveLinkButton(!sharedFolders?.some((f) => f.id === folderId));
+  }, [folderId, sharedFolders]);
+
+  if (!authenticated) {
+    return <Navigate to={routes.folder(folderId)} />;
+  }
+
+  if (loading) {
+    return <Spinner />;
+  } else if (error?.graphQLErrors[0]?.extensions?.status === 404) {
+    return <NotFound />;
+  } else if (error || !folder) {
+    return <ErrorPage />;
+  }
+
+  const resources = flattenResources(folder);
+
+  return (
+    <MyNdlaPageWrapper buttons={buttons} dropDownMenu={menu} viewType={viewType} onViewTypeChange={setViewType}>
+      <SharedFolder selectedFolder={folder} resources={resources} viewType={viewType} setViewType={setViewType} />
+    </MyNdlaPageWrapper>
+  );
+};
+
+export default FolderSharedPage;
+
+const Content = styled(ModalBody)`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing.normal};
+`;
+
+const ButtonRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  gap: ${spacing.small};
+  padding-top: ${spacing.large};
+`;
+
+interface SaveLinkProps {
+  folder: GQLFolder;
+  setShowTrigger: () => void;
+}
+
+const SaveLink = ({ folder, setShowTrigger }: SaveLinkProps) => {
+  const [open, setOpen] = useState(false);
+  const { t } = useTranslation();
+  return (
+    <Modal open={open} onOpenChange={() => setOpen(!open)}>
+      <ModalTrigger>
+        <ButtonV2 aria-label={t("myNdla.folder.sharing.button.saveLink")} variant="ghost">
+          <Subject />
+          {t("myNdla.folder.sharing.button.saveLink")}
+        </ButtonV2>
+      </ModalTrigger>
+      <SaveLinkContent setShowTrigger={setShowTrigger} folder={folder} onClose={() => setOpen(false)} />
+    </Modal>
+  );
+};
+
+interface SaveLinkContentProps extends SaveLinkProps {
+  onClose: VoidFunction;
+  setShowTrigger: () => void;
+}
+
+const SaveLinkContent = ({
+  folder: { id, name, subfolders, resources, status },
+  onClose,
+  setShowTrigger,
+}: SaveLinkContentProps) => {
+  const { saveSharedFolder } = useSaveFolderResourceMutation(id);
+
+  return (
+    <ModalContent>
+      <ModalHeader>
+        <ModalTitle>{t("myNdla.folder.sharing.save.header")}</ModalTitle>
+        <ModalCloseButton title={t("modal.closeModal")} />
+      </ModalHeader>
+      <ModalBody>
+        <Content>
+          <Folder
+            id={id}
+            title={name}
+            subFolders={subfolders.length}
+            subResources={resources.length}
+            link={routes.folder(id)}
+            isShared={status === "shared"}
+          />
+          <MessageBox>
+            <InformationOutline />
+            <Text margin="none">{t(`myNdla.folder.sharing.save.warning`)}</Text>
```

Comment:
Ingen praktisk betydning
```suggestion
            <Text margin="none">{t("myNdla.folder.sharing.save.warning")}</Text>
```